app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -447,6 +449,18 @@ class ConversationViewModel @Inject constructor(
         }
     }
 
+    private fun mapUserType(userType: UserType): Membership {
+        return when (userType) {
+            UserType.GUEST -> Membership.Guest
+            UserType.FEDERATED -> Membership.Federated
+            UserType.EXTERNAL -> Membership.External
+            UserType.INTERNAL -> Membership.None
+            else -> {
```

Comment:
Isn't `UserType` an `enum` ?

We don't need the `else` here in order to compile the code.

With the `else`, if we add new values in the future, this won't fail in compile time and the exception will be thrown unnecessarily.
I argue that handling new values in build time is better than having these silent errors crashing the app.